Client/mods/deathmatch/logic/luadefs/CLuaVehicleDefs.cpp

```diff
@@ -4052,3 +4056,42 @@ int CLuaVehicleDefs::OOP_GetVehicleModelExhaustFumesPosition(lua_State* luaVM)
     lua_pushboolean(luaVM, false);
     return 1;
 }
+
+std::variant<float, std::unordered_map<std::string, float>> CLuaVehicleDefs::GetVehicleModelWheelSize(
+    const unsigned short usModel, const std::optional<eResizableVehicleWheelGroup> eWheelGroup)
+{
+    eResizableVehicleWheelGroup eActualWheelGroup = eWheelGroup.value_or(eResizableVehicleWheelGroup::ALL_WHEELS);
+
+    if (eActualWheelGroup == eResizableVehicleWheelGroup::ALL_WHEELS)
+    {
+        float fFrontWheelSize;
+        if (!CStaticFunctionDefinitions::GetVehicleModelWheelSize(usModel, eResizableVehicleWheelGroup::FRONT_AXLE, fFrontWheelSize))
+            throw std::invalid_argument("Invalid model ID");
+
+        float fRearWheelSize;
+        if (!CStaticFunctionDefinitions::GetVehicleModelWheelSize(usModel, eResizableVehicleWheelGroup::REAR_AXLE, fRearWheelSize))
+            throw std::invalid_argument("Invalid model ID");
+
+        // Return a table like { ["front_axle"] = 0.7, ["rear_axle"] = 0.8 }
+        return std::unordered_map<std::string, float>{
+            {"front_axle", fFrontWheelSize},
+            {"rear_axle", fRearWheelSize},
+        };
+    }
+    else
+    {
+        float fWheelSize;
+        if (!CStaticFunctionDefinitions::GetVehicleModelWheelSize(usModel, eActualWheelGroup, fWheelSize))
+            throw std::invalid_argument("Invalid model ID");
+
+        return fWheelSize;
+    }
+}
+
+bool CLuaVehicleDefs::SetVehicleModelWheelSize(const unsigned short usModel, const eResizableVehicleWheelGroup eWheelGroup, const float fWheelSize)
+{
+    if (fWheelSize <= 0)
+        throw std::invalid_argument("Invalid wheel size");
+
+    return CStaticFunctionDefinitions::SetVehicleModelWheelSize(usModel, eWheelGroup, fWheelSize);
+}
```

Comment:
There is no need to use CStaticFunctionDefinitions in the new parser.

Reply:
Okay! But where do we put `CStaticFunctionDefinitions::GetVehicleModelWheelSize`, though?

Reply:
You can delete `CStaticFunctionDefinitions::GetVehicleModelWheelSize` and refactor the code, like this: https://pastebin.com/2J6qzHUM